test/visreg/visreg.ts

```diff
@@ -46,6 +45,18 @@ export function suite(browserName: string) {
         await snappit.snap("{browserName}/nav/custom-elements", $(util.selectors.nav));
     });
 
+    // Test Suite for Modal for Firefox browser
+    test("components/modals", async t => {
+        await util.go(driver, "components/modals");
+        await snappit.snap("{browserName}/modals/modals", $(util.selectors.modal));
+    });
+
+    test("components/open-modals", async t => {
+        await util.go(driver, "components/modals");
+        await driver.findElement(By.id('open-modal')).click();
+        await snappit.snap(`{browserName}/modals/open-modal`, snappit.$(util.selectors.modal));
```

Comment:
Please convert `snappit.$` to `$` like all other suite tests above.

Reply:
Replaced it with $